internal/core/item_rendering.rs

```diff
@@ -95,6 +96,43 @@ pub(crate) fn is_clipping_item(item: Pin<ItemRef>) -> bool {
         || ItemRef::downcast_pin::<Clip>(item).is_some()
 }
 
+fn render_item_children(
+    renderer: &mut dyn ItemRenderer,
+    component: &ComponentRc,
+    index: isize,
```

Comment:
That's an ItemRc.

Reply:
That's what I thought at first, too, but it turns out that's not the case for the very first call, where the index is `-1`. I could work around it by casting, but that feels more like a hack.